abstractions/src/Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))"/>
+  <PropertyGroup>
+    <Authors>OpenSearch Project and contributors</Authors>
+    <Copyright>OpenSearch</Copyright>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <RepositoryType>Git</RepositoryType>
+
+
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\build\keys\keypair.snk</AssemblyOriginatorKeyFile>
```

Comment:
The relative path is somewhat confusing, I'm assuming due to the nature of MSBuild, this actually gets evaluated at the actual project's context, meaning this does end up resolving to `abstractions/build/keys/keypair.snk`. 

Is there much reason this couldn't just use the main SNK as used by the clients, especially given this hasn't been truly published before as far as I'm aware?